werkzeug/contrib/jsrouting.py

```diff
@@ -18,9 +18,18 @@
         def dumps(*args):
             raise RuntimeError('simplejson required for jsrouting')
 
+import warnings
 from inspect import getmro
 from werkzeug.routing import NumberConverter
 from werkzeug._compat import iteritems
+from werkzeug.contrib import WerkzeugContribDeprecationWarning
+
+warnings.warn(
+    'werkzeug.contrib.jsrouting is deprecated as of version 0.15 and will'
```

Comment:
for this one i wonder if deprecation is the right path forward - are there users and/or replacements

but i'm `-0` so it probably could go

Reply:
From http://werkzeug.pocoo.org/docs/0.14/transition/#stop-using-deprecated-things, looks like it's already been called out as deprecated:

> stop using URL generation for JavaScript, it does not scale well with many public routing.

Reply:
thanks for that note